bridge/client/assets/default-branding/app-config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "appTitle": "keptn",
+  "appTitle": "Keptn",
```

Comment:
For this change, you also need to update the UI test here (which is failing right now): https://github.com/keptn/keptn/blob/a5dc071e72ce6d8d13e128bb1e48aa92fdd1ac2d/bridge/cypress/integration/app.spec.ts#L10

But this also changes our appearance and I'm not sure if this maybe should even stay `keptn`? Looping in @johannes-b

Reply:
When we are touching this piece, please go with `Keptn`.